mrjob/options.py

```diff
@@ -288,11 +288,11 @@ def add_emr_opts(opt_group):
         opt_group.add_option(
             '--aws-availability-zone', dest='aws_availability_zone',
             default=None,
-            help='Availability zone to run the job flow on'),
+            help='Availability zone to run EMR jobs in.'),
 
         opt_group.add_option(
             '--aws-region', dest='aws_region', default=None,
```

Comment:
We almost never set defaults in command-line options, since there's already a system for defaulting runner options (which can be set from either the command line or config file).

(Also there *isn't* a default for `aws_availability_zone`, default is to let the EMR API decide.)